auraed/src/cells/cell_service/cell_service.rs

```diff
@@ -39,6 +38,7 @@ use super::{
     },
     Result,
 };
+use crate::cells::cell_service::cells::CellsError;
```

Comment:
what's the difference between super::cells and crate::cells?

Reply:
`crate` is from the crate's root, and `super` is relative to the current module (i.e., `crate::cells::cell_service`). Now that we don't have `runtime`, we can probably flatten the structure a bit as I was also having trouble keeping the track of the module paths.